keps/sig-cluster-lifecycle/0035-20190128-addons-via-operators.md

```diff
@@ -0,0 +1,181 @@
+KEP: Addons via Operators
+
+---
+kep-number: 35
+title: Addons via Operators
+authors:
+  - "@justinsb"
+owning-sig: sig-cluster-lifecycle
+reviewers:
```

Comment:
We need some reviewers / approvers before we commit this. Maybe something to discuss tomorrow?

Reply:
If this has not been discussed in the SIG already it should be. The SIG should look at who the approvers are (the chairs?)

The related sigs field should also be filled in. I would suggest SIG Architecture as one of them.

Reply:
[This was raised to SIG Cluster Lifecycle in today's meeting]

Reply:
can someone post a summary of where the SIG is at on this?  There's no harm in merging and iterating.

Reply:
We can chat about it during our next call. 
/cc @justinsb

Reply:
Good point @mattfarina - I'd like to bring it to sig-architecture when we have something concrete to show them.

Reply:
SIG Arch is working to decentralize decision-making:
https://docs.google.com/document/d/1BlmHq5uPyBUDlppYqAAzslVbAO8hilgjqZUTaNXUhKM/edit#bookmark=id.7lj9uyhho0mu

I suggest discussing in SIG Cluster Lifecycle for now, and invite interested parties to attend.